File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples-java/ApiManagementGetPrivateLinkGroupResource.java
/** Samples for PrivateEndpointConnection GetPrivateLinkResource. */
public final class Main {
    /*
     * x-ms-original-file: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples/ApiManagementGetPrivateLinkGroupResource.json
     */
    /**
     * Sample code: ApiManagementGetPrivateLinkGroupResource.
     *
     * @param manager Entry point to ApiManagementManager.
     */
    public static void apiManagementGetPrivateLinkGroupResource(
        com.azure.resourcemanager.apimanagement.ApiManagementManager manager) {
        manager
            .privateEndpointConnections()
            .getPrivateLinkResourceWithResponse(
                "rg1", "apimService1", "privateLinkSubResourceName", com.azure.core.util.Context.NONE);
    }
}

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples-js/ApiManagementGetPrivateLinkGroupResource.js
const { ApiManagementClient } = require("@azure/arm-apimanagement");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets the private link resources
 *
 * @summary Gets the private link resources
 * x-ms-original-file: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/examples/ApiManagementGetPrivateLinkGroupResource.json
 */
async function apiManagementGetPrivateLinkGroupResource() {
  const subscriptionId = process.env["APIMANAGEMENT_SUBSCRIPTION_ID"] || "subid";
  const resourceGroupName = process.env["APIMANAGEMENT_RESOURCE_GROUP"] || "rg1";
  const serviceName = "apimService1";
  const privateLinkSubResourceName = "privateLinkSubResourceName";
  const credential = new DefaultAzureCredential();
  const client = new ApiManagementClient(credential, subscriptionId);
  const result = await client.privateEndpointConnectionOperations.getPrivateLinkResource(
    resourceGroupName,
    serviceName,
    privateLinkSubResourceName
  );
  console.log(result);
}
